app/src/main/java/swati4star/createpdf/util/PermissionsUtils.java

```diff
@@ -0,0 +1,57 @@
+package swati4star.createpdf.util;
+
+import android.app.Activity;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PermissionsUtils {
+    private static final String TAG = "PermissionsUtils";
+
+    /**
+     * checkRuntimePermissions takes in multiple permissions or an array of permissions
+     * and checks for if the permission is granted and if not it requests for the desired
+     * permissions. The try and catch block checks for invalid permissions passed.
+     *
+     * @param activity
```

Comment:
Edit these comments to include more description about function parameters and return type,